lxd/storage/backend_lxd.go

```diff
@@ -1972,7 +2042,9 @@ func (b *lxdBackend) RenameCustomVolume(volName string, newVolName string, op *o
 		oldName: volName,
 	})
 
-	err = b.driver.RenameVolume(drivers.VolumeTypeCustom, volName, newVolName, op)
+	vol := b.newVolume(drivers.VolumeTypeCustom, "", volName, nil)
```

Comment:
@monstermunchkin content type is hard coded to empty string but no comment explaining why, also no config.

Reply:
I missed that one.